.all-contributorsrc

```diff
@@ -11050,6 +11050,16 @@
         "code"
       ]
     },
+    {
+      "login": "ishubham010",
+      "name": "Shubham Choubey",
+      "avatar_url": "https://avatars.githubusercontent.com/u/71643172?v=4",
+      "profile": "https://github.com/ishubham010",
+      "contributions": [
+        "design",
+        "code"
+      ]
+    },
```

Comment:
do not rush, this is automacticaly generated by all-contributors plugin, so please remove it. It can be added later 💯

Reply:
I apologies for the mistake, Finally submitting clean code hope this time it will get commited.